nix/default.nix

```diff
@@ -0,0 +1,48 @@
+{ system ? builtins.currentSystem
+, crossSystem ? null
+, config ? {}
+, sourcesOverride ? {}
+}:
+let
+  sources = import ./sources.nix { inherit pkgs; }
+    // sourcesOverride;
+  iohKNix = import sources.iohk-nix {};
+  haskellNix = import sources."haskell.nix";
+  # use our own nixpkgs if it exists in our sources,
+  # otherwise use iohkNix default nixpkgs.
+  nixpkgs = if (sources ? nixpkgs)
+    then (builtins.trace "Not using IOHK default nixpkgs (use 'niv drop nixpkgs' to use default for better sharing)"
+      sources.nixpkgs)
+    else (builtins.trace "Using IOHK default nixpkgs"
```

Comment:
I'd prefer not to be spammed by one of these messages every time I build something!

Reply:
Let's just drop `nixpkgs` from the sources then? :-)

Reply:
It traces in that case too! Unconditional trace messages! Deleting the one for the "normal" case would help...

We probably can go back to the iohk nixpkgs now they've finally moved on to 19.09.